stubs/app/Orchid/Layouts/User/UserListLayout.php

```diff
@@ -54,7 +54,7 @@ public function columns(): array
                     return $user->updated_at->toDateTimeString();
                 }),
 
-            TD::set('id', 'ID')
+            TD::set('actions', 'Actions')
```

Comment:
Hey. There is no value binding, which means we can specify a record without a second argument.
```php
TD::set(__('Actions'))
```
It would also be great if we wrapped the string as translation

Reply:
@tabuna added string to the translation.